docsrc/source/pages/use_cases/comparing_datasets.rst

```diff
@@ -42,11 +40,9 @@ In order to compare more than two reports, the following syntax can be used:
     # Save report to file
     comparison_report.to_file("comparison.html")
 
-Note that generating reports for three or more datasets is not (yet) fully supported.
+Note that this functionality only ensures the support to generate reports to compare two datasets.
```

Comment:
I think the proposed wording is harder to read than the original.

Reply:
We won't include the (yet) fully supported. Hence the change.